index.html

```diff
@@ -1,22 +1,30 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ja">
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" type="text/css" />
-    <title>TerraceTechフロントエンドエンジニア養成所（修正済みです！）</title>
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
   </head>
   <body>
     <div>
-      <ul id="js-list"></ul>
+      <ul id="js-list">
+      </ul>
     </div>
-    <script>
-      const ul = document.getElementById('js-list');
-      const li = document.createElement('li'); const a = document.createElement('a'); const img = document.createElement('img');
-      a.setAttribute("href","1.html"); img.setAttribute("src","bookmark.png"); img.setAttribute("alt","ブックマーク"); 
-      a.textContent = 'これです';
-      ul.appendChild(li);  li.appendChild(a); a.appendChild(img);
+    <script>    
+      var ul = document.getElementById('js-list');
+      for(var i=1; i<=2; i++) {
+      var li = document.createElement('li');
+      li.textContent = 'a'+i;
+      ul.appendChild(li);
+      var a = document.createElement('a');
+      a.href= 'a'+i+'.html'
```

Comment:
You can also write it this way.
```
a.href = `a${i}.html`;
```
https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Template_literals

Reply:
Thank you for telling me a different method.
I thought it was simple and very good, so I'll fix it.